src/main/java/lotto/LottoApplication.java

```diff
@@ -0,0 +1,17 @@
+package lotto;
+
+import lotto.model.LottoGenerator;
+import lotto.view.LottoInputView;
+import lotto.view.LottoOutputView;
+
+public class LottoApplication {
+
+    public static void main(String[] args) {
+        LottoGame game = new LottoGame(
+            new LottoGenerator()
+            , new LottoInputView()
+            , new LottoOutputView()
+        );
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break
```suggestion
        LottoGame game = new LottoGame(
            new LottoGenerator(),
            new LottoInputView(),
            new LottoOutputView()
        );
```